include/private/vkd3d_atomic.h

```diff
@@ -108,7 +108,8 @@ FORCEINLINE uint32_t vkd3d_atomic_uint32_decrement(uint32_t *target, vkd3d_memor
     return result;
 }
 
-FORCEINLINE uint32_t vkd3d_atomic_uint32_compare_exchange(uint32_t* target, uint32_t expected, uint32_t desired, vkd3d_memory_order order)
+FORCEINLINE uint32_t vkd3d_atomic_uint32_compare_exchange(uint32_t* target, uint32_t expected, uint32_t desired,
```

Comment:
This breaks MSVC build, you should pass success_order to where order is in chooseintrinsic

Reply:
How?

Reply:
Ah, yeah, think I get what you mean. Should be fixed, but I can't really test MSVC builds right now.